images/ingress-nginx-controller/config/latest.apko.yaml

```diff
@@ -0,0 +1,121 @@
+contents:
+  packages: [
+    # Package comes from extra-packages
+  ]
+
+accounts:
+  groups:
+    - groupname: www-data
+      gid: 101
+  users:
+    - username: www-data
+      uid: 101
+  run-as: 101
+
+environment:
+  PATH: "$PATH:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/luajit/bin:/usr/local/nginx/sbin/"
+  LUA_PATH: "/usr/local/share/lua/5.1/?.lua;/usr/local/lib/lua/?.lua;/usr/lib/lua/?.lua;/usr/share/lua/?.lua"
+  LUA_CPATH: "/usr/local/lib/lua/5.1/?.so;/usr/local/lib/lua/?/?.so;/usr/local/lib/lua/?.so;/usr/lib/lua/?.lua;"
+  LD_PRELOAD: "/usr/lib/libmimalloc.so.1"
```

Comment:
These paths are using /usr/local it looks like the lua packages in wolfi are not specifying a PREFIX=/usr in many if not all lua packages. This should probably be fixed and the paths above updated.

Reply:
We can either merge this and I can followup with the lua updates else happy to hold this and merge all together.